lebedigital/demonstrator_scripts/computation_sc_volume_fraction.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from lebedigital.unit_registry import ureg
+
+
+def computation_sc_volume_fraction(input_dic):
+    # initialize output dictionary
+    output = {}
+
+    if input_dic["sc_mass_fraction"] == 0:
+        return 0
+    else:
+        mass_sub = 1 * ureg("kg") / (1 + (1 - input_dic["sc_mass_fraction"]) / input_dic["sc_mass_fraction"])
```

Comment:
not sure I can follow this formula, is that the mass of cement substitute per m³ of concrete?